javascript/code-challenges/challenges-02.test.js

```diff
@@ -0,0 +1,305 @@
+'use strict';
+
+const { map } = require("cheerio/lib/api/traversing");
+
+/* ------------------------------------------------------------------------------------------------
+
+CHALLENGE 1 - Review
+
+Write a function named raisedToTheThird that takes in an array of numbers and returns a new array of each of those numbers raised to the 3rd power (hint: look up Math.pow()). Use forEach to solve this problem.
+
+------------------------------------------------------------------------------------------------ */
+
+const raisedToTheThird = (arr) => {
+  let output = [];
+  arr.forEach(value => output.push(Math.pow(value, 3)));
```

Comment:
You can also use the exponent "**" sign to do this. Using "**" actually is just short hand for invoking Math.pow().
```js
value ** 3;
```